pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: default
+---
+
+# Single Line Diagram - SVG Writing
+
+The `com.powsybl.sld.svg.DefaultSVGWriter` class of [powsybl-single-line-diagram](../repositories/powsybl-single-line-diagram.md) is an implementation of `SVGWriter` interface which allows to generate SVG files representing single line diagrams of a substation, a voltage level or a zone. 
+We are showing in this guide how to create some single line diagrams
+- first from a test network,
+- then from a CGMES file.
+
+*Note: the 1.8.0-SNAPSHOT version of powsybl-single-line-diagram was used to generate the SVGs of this guide. Because of that, you might experience some differences with the examples shown, which will be corrected by 1.8.0 release (expected: 2020-11-26).*
```

Comment:
Remove this sentence and add the `do-not-merge` label. I think it's better to wait for the next release instead of talking about a development version.